packages/components/tsconfig.json

```
@@ -0,0 +1,26 @@
+{
```

As discussed IRL, I'd love to eventually

```suggestion
{
"extends": "@automattic/calypso-build/tsconfig",
```

Not a blocker though